src/tests/integration/konvert/test_simple_proofs.py

```diff
@@ -362,3 +377,30 @@ def test_kore_to_kast(
 
         # Then
         assert actual_kast == kast
+
+    @pytest.mark.parametrize(
+        'rule_id,kore_text',
+        KRULE_TO_KORE_DATA,
+        ids=[rule_id for rule_id, *_ in KRULE_TO_KORE_DATA],
+    )
+    def test_krule_to_kore(
+        self,
+        kprint: KPrint,
+        rule_id: str,
+        kore_text: str,
+    ) -> None:
+        k_definition = kprint.definition
+        ms = [m for m in k_definition.modules if m.name == k_definition.main_module_name]
+        assert len(ms) == 1
+        mm = ms[0]
+
+        rules = [r for r in mm.rules if (('label' in r.att) and (r.att['label'] == rule_id))]
+        assert len(rules) == 1
+        r = rules[0]
```

Comment:
`from pyk.utils import single`, then

```suggestion
        r = single(r for r in mm.rules if 'label' in r.att and r.att['label'] == rule_id)
```

Reply:
Consider typing this out as well, `rule` is more readable than `r`.